### PR TITLE
UI bug fixes for services overview section

### PR DESCRIPTION
**Pull request by SauarvD**
### Requirements

* To make Events Section editable
* UI Bug fixes

### Description of the Change
* CSS fixes are done for making the message prettier and for the radio button to be centred across the application as well.
* Functionality and Script changes are done for enabling the edit button in the events section.

### Benefits

* Now the user can enable the cron schedule back through the UI once he/she disables the schedule.

### CODE REVIEW

File: core/jazz_ui/src/app/pages/service-overview/service-overview.component.ts [hunk 1]
```diff
@@ -371,42 +377,24 @@ export class ServiceOverviewComponent implements OnInit {
     this.saveClicked = false;
     this.advancedSaveClicked = true;
     let payload = {};
+    let obJ = {};
```
Comment (supritat) [inline core/jazz_ui/src/app/pages/service-overview/service-overview.component.ts]: Can you rename this var as 'eventObj'?

File: core/jazz_ui/src/app/pages/service-overview/service-overview.component.ts [hunk 2]
```diff
@@ -443,8 +431,14 @@ export class ServiceOverviewComponent implements OnInit {
   }
 
   onCancelClick() {
+    if(this.service.eventScheduleEnable === false){
+      this.rateExpression.type = "none"
+    } else {
+      this.rateExpression.type = "cron"
```
Comment (supritat) [inline core/jazz_ui/src/app/pages/service-overview/service-overview.component.ts]: try not to hard code 'this.rateExpression.type' value. assign based on the radio button selected.

File: core/jazz_ui/src/app/pages/service-overview/service-overview.component.html [hunk 3]
```diff
@@ -308,13 +309,18 @@
   <div class='confirm-overlay' (click)='onAdvanceClick()' *ngIf='saveClicked || advancedSaveClicked'>
     <div class='confirm-overlay-inner-wrap' (click)="$event.stopPropagation()">
         <div class='overlay-header'>Submitting Changes</div>
-        <div *ngIf="isPUTLoading" class='loader-confirmation'>
+        <!-- <div *ngIf="isPUTLoading" class='loader-confirmation'>
```
Comment (supritat) [inline core/jazz_ui/src/app/pages/service-overview/service-overview.component.html]: Remove commented or unused code.

File: core/jazz_ui/src/app/pages/service-overview/service-overview.component.scss [hunk 4]
```diff
@@ -1277,8 +1284,12 @@ details {
     }
     .overlay-body {
         text-align: center;
-        padding: 3rem;
+        // padding: 3rem;
```
Comment (supritat) [inline core/jazz_ui/src/app/pages/service-overview/service-overview.component.scss]: Remove if not being used any more.

File: core/jazz_ui/src/app/pages/service-overview/service-overview.component.ts (hunk 1 above)
Comment (SauarvD) [reply core/jazz_ui/src/app/pages/service-overview/service-overview.component.ts]: renamed the var as 'eventObj'

File: core/jazz_ui/src/app/pages/service-overview/service-overview.component.html (hunk 3 above)
Comment (SauarvD) [reply core/jazz_ui/src/app/pages/service-overview/service-overview.component.html]: Removed the commented code

File: core/jazz_ui/src/app/pages/service-overview/service-overview.component.scss (hunk 4 above)
Comment (SauarvD) [reply core/jazz_ui/src/app/pages/service-overview/service-overview.component.scss]: Removed the commented line

File: core/jazz_ui/src/app/pages/service-overview/service-overview.component.ts (hunk 2 above)
Comment (SauarvD) [reply core/jazz_ui/src/app/pages/service-overview/service-overview.component.ts]: Assigned the rateExpression type value according to the service value

**Pull request merged**